brain_agents_orchestrator.py

```
@@ -304,13 +304,15 @@ def run_agentic_loop(self, scalper_instance, symbol="EURUSD"):
                 try:
                     res = fut.result()
                     method_scores[res["method"]] = res["score"]
-                except Exception: pass
+                except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)